swatch-producer-aws/src/main/java/com/redhat/swatch/aws/exception/UsageTooOldException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.aws.exception;
+
+import static com.redhat.swatch.aws.exception.ErrorCode.USAGE_TOO_OLD;
+
+public class UsageTooOldException extends AwsProducerException {
```

Comment:
I would add a javadoc with a link to https://docs.aws.amazon.com/marketplacemetering/latest/APIReference/API_UsageRecord.html and say that the "Timestamp" property is what matters here. 

Moreover, I would rename this class "UsageTooOldException" to "UsageTimestampOutOfBoundsException" or something like this. Yet, if you would like to keep using "UsageTooOldException", I'm ok too.

Reply:
Solid suggestions, I'll make the changes. Thanks!

Reply:
Done.